docs/standard/modern-web-apps-azure-architecture/develop-asp-net-core-mvc-apps.md

```diff
@@ -552,6 +553,5 @@ _Learn more about Azure deployment options in [Chapter 10](development-process-f
 > - **Introducing Azure Application Gateway**  
 >   <https://docs.microsoft.com/azure/application-gateway/application-gateway-introduction>
 
->[!div class="step-by-step"]
->[Previous](common-client-side-web-technologies.md)
->[Next](work-with-data-in-asp-net-core-apps.md)
\ No newline at end of file
+> [!div class="step-by-step"][previous](common-client-side-web-technologies.md)
```

Comment:
The changes on lines 556 and 557 should be reverted since they've broken the navigation:

![image](https://user-images.githubusercontent.com/10886961/50660019-38228780-0f53-11e9-8538-b22b94819a3a.png)

Reply:
Done - sorry about that. Was automatic format-on-save with VS Code and a MD linter.